src/components/Chat/InputMessageV2.tsx

```diff
@@ -97,7 +116,7 @@ const styles = StyleSheet.create({
 });
 
 export interface InputMessageV2Props {
-  onSendButtonClicked: (message: string) => void;
+  onSendButtonClicked: (message: string, attachments: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSendButtonClicked` function now accepts an additional parameter `attachments`. This is a breaking change and could potentially cause issues in parts of the code where this function is used but has not been updated to include the new parameter. Please ensure that all usages of this function have been updated accordingly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->